src/Hl7.Fhir.Base/FhirPath/Parser/Lexer.cs

```diff
@@ -68,7 +68,7 @@ internal partial class Lexer
 
         public static readonly Parser<string> DelimitedIdentifier =
 
-            DelimitedContents('"')
+            DelimitedContents('"') // this isn't to standard
```

Comment:
It was there for backwards compat. Should we axe it?

Reply:
I'm OK with it remaining for that purpose. Maybe I'll just update the code comment to say that's why it's there?